activerecord/test/cases/unsafe_raw_sql_test.rb

```diff
@@ -167,11 +167,9 @@ class UnsafeRawSqlTest < ActiveRecord::TestCase
   end
 
   test "order: logs deprecation warning for unrecognized column" do
-    e = assert_raises(ActiveRecord::UnknownAttributeReference) do
+    assert_raises(ActiveRecord::UnknownAttributeReference) do
       Post.order("REPLACE(title, 'misc', 'zzzz')")
     end
-
-    assert_match(/Query method called with non-attribute argument\(s\):/, e.message)
```

Comment:
You could assert the message:

`Dangerous query method (method whose arguments are used as raw SQL) called with non-attribute argument(s):`

Reply:
OK